src/makeCalculator.js

```diff
@@ -37,7 +37,31 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+    add(result, num) {
+      return result + num;
+    },
+    subtract(result, num) {
```

Comment:
```suggestion
    result: 0,
    
    add(result, num) {
      return result + num;
    },
    
    subtract(result, num) {
```
add spaces between properties